### PR TITLE
[WASM][AOT] Do not include KernelTraceControl in WasmAssembliesToBundle

### PR DESCRIPTION
**Pull request by naricc**
KernelTraceControl.dll keeps showing up in the publish directory, where the AOT compiler then tries to compile it. But it can't, since its a native dll.

The long term fix for this is probably to have the compiler skip any native assemblies it encounters, but that is not ready yet.

Fixes: https://github.com/dotnet/performance/issues/2026

### CODE REVIEW

**Comment by naricc** [comment]
@adamsitnik

**Comment by naricc** [comment]
@AndreyAkinshin Any chance you could review and merge this? I notice Adam is on vacation.

File: src/BenchmarkDotNet/Templates/WasmAotCsProj.txt [hunk 1]
```diff
@@ -37,8 +38,9 @@
     <WasmBuildAppAfterThisTarget>PrepareForWasmBuild</WasmBuildAppAfterThisTarget>
   </PropertyGroup>
   <Target Name="PrepareForWasmBuild" AfterTargets="Publish">
+    <Message Text="!!!debug!!! TargetDir: $(TargetDir)" />
```
Comment (AndreyAkinshin) [inline src/BenchmarkDotNet/Templates/WasmAotCsProj.txt:41]: Do we actually need this `!!!debug!!!` message?

Comment (naricc) [reply src/BenchmarkDotNet/Templates/WasmAotCsProj.txt]: Whoops, removed it.

**Review by AndreyAkinshin — APPROVED** [review]

**Comment by AndreyAkinshin** [comment]
@naricc thanks!

**Pull request merged**